types/qaengine/problem.go

```diff
@@ -39,7 +39,7 @@ const (
 	// InputSolutionFormType allows single line user input
 	InputSolutionFormType SolutionFormType = "Input"
 	// MultilineInputSolutionFormType allows multiple user input
-	MultilineInputSolutionFormType SolutionFormType = "MultiLineInput"
+	MultilineInputSolutionFormType SolutionFormType = "MultiLine"
```

Comment:
why do we need to change this?

Reply:
@HarikrishnanBalagopal 
https://github.com/konveyor/move2kube-ui/pull/128#discussion_r980712703